src/cmd-buildextend-metal

```diff
@@ -172,8 +172,16 @@ if [ -z "${use_anaconda}" ]; then
     if [ -n "${ref_is_temp}" ]; then
         ref_arg=${commit}
     fi
-
-    runvm -drive "if=virtio,id=target,format=${image_format},file=${path}.tmp" -- /usr/lib/coreos-assembler/create_disk.sh /dev/vda "$ostree_repo" "${ref_arg}" "${ostree_remote}" /usr/lib/coreos-assembler/grub.cfg "$name" "${save_var_subdirs}" "\"$kargs\""
+    runvm -drive "if=virtio,id=target,format=${image_format},file=${path}.tmp" -- \
+            /usr/lib/coreos-assembler/create_disk.sh \
+                -d /dev/vda \
```

Comment:
If we're doing this to improve legibility, then let's use the long forms?

Reply:
Long forms _are_ my preference as well. But I've seen the short-forms used in this codebase. Happy to make that change.

Reply:
Done.